frontend/packages/kubevirt-plugin/integration-tests/tests/models/wizard.ts

```diff
@@ -97,13 +93,21 @@ export class Wizard {
   }
 
   async selectWorkloadProfile(workloadProfile: string) {
-    await selectOptionByText(view.workloadProfileSelect, workloadProfile);
+    await selectItemFromDropdown(view.workloadProfileSelect, view.dropDownItem(workloadProfile));
   }
 
   async selectProvisionSource(provisionOptions) {
-    await selectOptionByText(view.provisionSourceSelect, provisionOptions.method);
+    await selectItemFromDropdown(
+      view.provisionSourceSelect,
+      view.bootSourceDropDownItem(provisionOptions.method),
```

Comment:
can we call it provisionSourceDropDownItem for consistency reasons? I know we renamed it in the UI. but we should be consistent here IMO.

Or we should rename all the occurrences here

Reply:
I used the code listed here:
```
async selectOperatingSystem(operatingSystem: string) {
    await selectItemFromDropdown(view.operatingSystemSelect, view.dropDownItem(operatingSystem));
  }
```
and it couldn't select OS when running VMWare wizard

Reply:
> can we call it provisionSourceDropDownItem for consistency reasons? I know we renamed it in the UI. but we should be consistent here IMO.
ok.
> and it couldn't select OS when running VMWare wizard
I guess the style of OS list in VMWare wizard is different from vm Wizard, if so, I think the best way is making the style consistent.

Reply:
It is using the same component in the code. @ibragins can you check if it renders different html/styles for you?

Reply:
done